### ISSUE TITLE
Fix typos by introducing whitespaces

### ISSUE THREAD

**Issue by littlewine** [barrust/pyspellchecker#74]
Is there a way (or workaround) to correct mistakes (typos) also by introducing whitespaces?

eg. "yes specificallyshows [...]" -> "yes specifically shows"

I guess it could be somewhat more complicated, since you have to match with two words, split accordingly, etc.

**Comment by barrust** [comment]
You are correct that it would be more complicated but possible. It may make more sense for that to be outside the library itself and more of an as needed solution. If you come up with a method and think it would be a good addition to the library, I accept Pull Requests!
